.gitignore

```diff
@@ -8,3 +8,6 @@ proto.pb
 # IntelliJ
 .idea/
 *.iml
+
+# gofmt
+*.grc.bk
```

Comment:
Is this actually coming from Go Report Card? (grc)[ ](https://github.com/gojp/goreportcard/blob/9eb5ac8a26ec6ef7e03bca667d5a565cb82f8b98/check/utils.go#L71)